irods/manager/zone_manager.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+import logging
+
+from irods.models import Zone
+from irods.zone import iRODSZone
+from irods.manager import Manager
+from irods.message import GeneralAdminRequest, iRODSMessage
+from irods.api_number import api_number
+
+logger = logging.getLogger(__name__)
+
+class ZoneManager(Manager):
+
+    def get(self, zone_name):
+        query = self.sess.query(Zone).filter(Zone.name == zone_name)
+
+        try:
+            result = query.one()
+        except NoResultFound:
+            raise UserDoesNotExist()
```

Comment:
`UserDoesNotExist`?   is there something better?   should there be?

Reply:
yes, should be ZoneDoesNotExist. Just a cut-paste error